### PR TITLE
Cookie policy window

### PR DESCRIPTION
**Pull request by pawelczak**
#626

### CODE REVIEW

File: saos-webapp/src/main/resources/message/common.properties [hunk 1]
```diff
@@ -83,6 +83,13 @@ month.11 = november
 month.12 = december
 
 
+/* Cookie policy */
+cookie.policy.previewMessage = Polityka cookies
+cookie.policy.show = Zobacz
+cookie.policy.message = Ta strona wykorzystuje pliki cookies (niewielkie pliki tekstowe przechowywane przez przeglądarkę internetową na urządzeniu użytkownika) m.in. do analizy statystycznej ruchu, dopasowania wyglądu i treści strony do indywidualnych potrzeb użytkownika. Pozostawiając w ustawieniach przeglądarki włączoną obsługę plików cookies wyrażasz zgodę na ich użycie. Jeśli nie zgadzasz się na wykorzystanie plików cookies zmień ustawienia swojej przeglądarki.
```
Comment (lukdumi) [inline saos-webapp/src/main/resources/message/common.properties]: What do you mean by "statystyczna analiza ruchu"? Future analytics tools like google analythics for example?

File: saos-webapp/src/main/webapp/WEB-INF/static/javascript/modules/cookiePolicy.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        initButtonShowCookiePolicy();
+        initButtonAcceptCookiePolicy();
+        
+    };
+    
+    return scope;
+    
+    
+    /*
+     * Show cookie policy message
```
Comment (lukdumi) [inline saos-webapp/src/main/webapp/WEB-INF/static/javascript/modules/cookiePolicy.js]: While describing methods the 3rd person form of a verb should be used. Here: Shows

File: saos-webapp/src/main/webapp/WEB-INF/static/javascript/modules/cookiePolicy.js [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+            $(options.windowId).fadeOut();
+            
+            //Show cookie policy 
+            $(options.messageId).slideDown();
+        });
+
+    }
+    
+    /*
+     * Create cookie & hide message
```
Comment (lukdumi) [inline saos-webapp/src/main/webapp/WEB-INF/static/javascript/modules/cookiePolicy.js]: Creates

**Comment by lukdumi** [comment]
I am not sure if the solution with a small box is the best one. A user has to click twice to close the cookie info. Maybe the little box at the beginning should have an accept button too?

File: saos-webapp/src/main/resources/message/common.properties (hunk 1 above)
Comment (pawelczak) [inline saos-webapp/src/main/resources/message/common.properties]: Yes.

**Pull request closed without merging**